manifests/charts/istiod-remote/values.yaml

```
@@ -415,3 +415,6 @@ base:
   # it works to have multiple redundant validations, this adds complexity and operational risks.
   # Users should consider enabling this if they want full gateway-api validation but don't have other validation servers.
   validateGateway: false
+# set to true if the corresponding istio-cni chart has been installed
+istio_cni:
+  enabled: false
```

```suggestion
  enabled: false
  chained: false

```

It looks like this should also be added.

cc @rrobb-aspen